src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag

```diff
@@ -44,8 +44,8 @@
             <tbody>
                 <tr>
                     <th class="text-center">User/Group</th>
-                    <th class="text-center">Can see your comment</th>
-                    <th class="text-center">Can see your name</th>
+                    <th class="text-center">Can see this comment</th>
+                    <th class="text-center">Can see giver's name</th>
```

Comment:
Will this be confusing because there are two givers here: feedback response giver and comment giver?